presto-iceberg/src/test/java/com/facebook/presto/iceberg/hive/TestIcebergDistributedHive.java

```diff
@@ -26,4 +35,24 @@ public TestIcebergDistributedHive()
     {
         super(HIVE);
     }
+
+    @Override
+    protected Table loadTable(String tableName)
```

Comment:
Would you mind telling me a bit more about the reasons for overriding the `loadTable` function? I noticed that you override the function in the `TestIcebergDistributedHive` and `TestIcebergDistributedNessie`, but not in the `hadoop` test.

Reply:
`loadTable` actually uses Iceberg's API to load the table. For `TestIcebergDistributedHive`, we need to specify the hms address, but we use `FileHiveMetastore` here. Iceberg does not support this, so the `loadTable` implementation is rewritten in `TestIcebergDistributedHive` and uses Presto's API to load Iceberg table.  For `TestIcebergDistributedNessie`, because the url of the Nessie service is required, `getProperties` is rewritten in it to specify the url of the Nessie service. For `TestIcebergDistributedHadoop`, it only needs to specify the `warehouse` attribute to load the Iceberg table, so there is no need to rewrite.